src/osd/osd_types.cc

```diff
@@ -1510,7 +1509,7 @@ void pg_pool_t::encode(bufferlist& bl, uint64_t features) const
     ::encode(removed_snaps, bl);
     ::encode(auid, bl);
     ::encode(flags, bl);
-    ::encode(crash_replay_interval, bl);
+    ::encode((uint32_t)(crush_rule == 0 && auid == 0 ? 60 : 0), bl); // crash_replay_interval
```

Comment:
What's this odd math about? It looks like you copied it from an old upgrade path, but that was trying to deal with when we *didn't* have the crash replay interval and needed to default set it for proper behavior. Now the value is meaningless. So just set it to zero (or a magic value) and let it be a reserved field for the future?

Reply:
oh, right.  i was thinking we needed to put something meaningful in there for compat with old clients but only osds every looked at this and they're >= luminous now.